armi/tests/detailedAxialExpansion/refSmallCoreGrid.yaml

```diff
@@ -2,12 +2,34 @@ core:
   geom: hex
   symmetry: third periodic
   lattice map: |
-         -  PC
-          IC  IC
-            IC  OC
-          IC  LA  IC
-            IC  IC  SC
-              LA  IC
-            IC  IC  OC
-              IC  IC
-            AF  IC  OC
+         -  SH
+          MC  SH
+            MC  SH
+          MC  MC  SH
+            AF  MC  SH
+              AF  MC
+            IC  LA  MC
+              IC  LA
+            PC  IC  MC
+
+fourPin:
+  geom: hex_corners_up
+  symmetry: full
+  lattice map: |
+    - - - - - - - - 1 1 1 1 1 1 1 1 1
```

Comment:
This seems like a really good time to reduce the size of the axial expansion test reactor.

Surely we could test this feature with fewer rings?

Reply:
reduce the size of the core map or this block grid?

Reply:
I mean... both in an ideal world.

These are unit tests, they should be fast and small.

But doing either would make the tests faster, so "perfect is the enemy of better", I'll take either.

Reply:
@john-science https://github.com/terrapower/armi/pull/2145/commits/a27f0259318b6f7de5539f226b865bd87faacfab has a _much_ smaller pin lattice that works

Reply:
@john-science https://github.com/terrapower/armi/pull/2145/commits/3e6c1d5feac0835f313e91c74e7e73c4de581415 has as small of a assembly lattice as I think we're going to get!